code/datums/mutantraces.dm

```diff
@@ -2042,7 +2044,9 @@ TYPEINFO(/datum/mutantrace)
 
 	say_filter(var/message)
 		.= replacetext(message, "cow", "human")
-		.= replacetext(., "m", stutter("mm"))
+		var/replace_lowercase = replacetextEx(message, "m", stutter("mm"))
+		var/replace_uppercase = replacetextEx(replace_lowercase, "M", stutter("MM"))
+		return replace_uppercase
```

Comment:
pretty sure this breaks the 'cow' -> 'human' replacement, since that's using `.` to store the replacement, while the rest of the stuff is operating on `message`. Suggestion will fix.
```suggestion
		var/replace_lowercase = replacetextEx(., "m", stutter("mm"))
		var/replace_uppercase = replacetextEx(replace_lowercase, "M", stutter("MM"))
		return replace_uppercase
```

Reply:
basically the cow -> human thing gets stored in `.` but that's never referenced again so it just kinda disappears